requirements-app.txt

```
@@ -19,12 +19,13 @@ gunicorn==20.0.4
 eventlet==0.29.1
 notifications-python-client==5.7.0
 Shapely==1.7.1
+xlrd==1.2.0  # 2.0.1 is incompatible 
```

we can make sure pyup doesn't touch anything

```suggestion
xlrd==1.2.0  # pyup: <2 # 2.0.1 drops xlsx support
```